aspnetcore/security/authentication/azure-ad-b2c-webapi.md

```diff
@@ -0,0 +1,208 @@
+    
+    ![Authorization tab with settings](./azure-ad-b2c-webapi/postman-auth-tab.png)
+
+2. Complete the **GET NEW ACCESS TOKEN** dialog as follows:
+    
+    | Setting                   | Value                                                                                         | Notes                                                                                                  |
+    |---------------------------|-----------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|
+    | **Token Name**            | *&lt;token name&gt;*                                                                          | Enter a descriptive name for the token.                                                                |
+    | **Grant Type**            | Implicit                                                                                      |                                                                                                        |
+    | **Callback URL**          | `https://getpostman.com/postman`                                                              |                                                                                                        |
+    | **Auth URL**              | `https://login.microsoftonline.com/<tenant domain name>/oauth2/v2.0/authorize?p=B2C_1_SiUpIn` | Be sure to replace *&lt;tenant domain name&gt;* with the tenant's domain name with no angle brackets.  |
+    | **Client ID**             | *&lt;enter the Postman app's <b>Application ID</b>&gt;*                                          |                                                                                                        |
+    | **Client Secret**         | *&lt;leave blank&gt;*                                                                         |                                                                                                        |
+    | **Scope**                 | `https://<tenant domain name>/api/user_impersonation openid offline_access`                   | Be sure to replace *&lt;tenant domain name&gt;* with the tenant's domain name with no angle brackets.  |
+    | **Client Authentication** | Send client credentials in body                                                               |                                                                                                        |
+    
+3. Select the **Request Token** button.
+
+4. Postman opens a new window containing the Azure AD B2C tenant's sign in dialog. Sign in with an existing account (if one was created testing the policies) or select **Sign up now** to create a new account. The **Forgot your password?** link is used to reset a forgotten password.
```

Comment:
If your pattern is to comma the conjunction (consistency), then ... "... the policies), or select ..."